ui/app/AppLayouts/Wallet/panels/SearchableCollectiblesPanel.qml

```diff
@@ -76,9 +94,15 @@ Control {
             StatusListView {
                 id: collectiblesListView
 
+                readonly property bool validSearchResultExists: !!collectiblesSearchBox.text && sfpm.rowCount() > 0
```

Comment:
```suggestion
                readonly property bool validSearchResultExists: !!collectiblesSearchBox.text && sfpm.count > 0
```

Reply:
The `rowCount()` is a function and thus the binding won't get updated

Reply:
if the used modal is known in given context to be e.g. ListModel or SFPM, we can use `count`, when we handle generic model from outside, the proper way is to use generic `ModelCount.count`.